.github/workflows/pr-release.yaml

```diff
@@ -8,7 +8,7 @@ jobs:
     name: "Release PR to npm"
     runs-on: ubuntu-latest
     # comment out if:false to enable release PR to npm
-    if: false
+    # if: false
```

Comment:
Should we uncomment this `if`?

Reply:
Only at the very last moment, but yes.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1291/commits/330ec509ba8dd0c2503bb3a7a6fa0e57ac88ac41